extensions/wikia/WikiFactory/tests/WikiFactoryTest.php

```diff
@@ -223,6 +241,13 @@ public function testPrepareUrlToParseDataProvider() {
 		];
 	}
 
+	public function testGetHostById() {
+		$this->mockStaticMethod( 'WikiFactory', 'getVarValueByName', 1 );
+		$this->mockStaticMethod( 'WikiFactory', 'getLocalEnvURL', 'test_host/' );
+
+		$this->assertEquals( 'test_host', WikiFactory::getHostById( 2 ) );
```

Comment:
Are we checking `rtrim` here? :)

Why do we need to call rtrim in `getHostById`, but not in `getLocalEnvURL`?